examples/USBH_MIDI/bidirectional_converter/bidirectional_converter.ino

```diff
@@ -87,6 +88,9 @@ void loop()
           //SysEx is handled by event.
           break;
         default :
+          if( msg[0] < 0xf0 ){
+            msg[0] |= MIDI.getChannel() - 1;
+          }
```

Comment:
Could you add a comment explaining why this code is needed, so people less familiar with MIDI (including me) would be able to understand it?

Reply:
- msg [0] is the MIDI status byte.
- Lower 4 bits of the status byte is the channel number.
- Status byte 0xFn is a system message.
- System messages do not have a channel number.

I forgot to set the channel number.
The getChannel() function in the Arduino MIDI library returns 1-16 as channel number.
https://fortyseveneffects.github.io/arduino_midi_library/a00033.html#ga0d2a0037fd9e4b06cafe7b2388451767
However, the channel number of raw MIDI messages is zero origin.

Reply:
Okay. Can you add that comment to the example? Then I'll merge it :)